### PR TITLE
Fixing frontend documentation as per Katz

### PR DESCRIPTION
**Pull request by stefanhts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,38 +10,37 @@ be able to see their problem question and how many tests they are passing. Once
...
-3. Ensure your machine has the ability to open html files in browser.
-4. Right click on extension.js.html and select the option "Open In Default Browser".
-   or however you wish to open the html file.
+## Using the Extension
+1. Install the `.vsix` extension file into your instance of VSCode.
+    a. Navigate to the extensions tab on the left of the window.
+    b. Click the `...` in the upper right
+    c. Select "install from vsix"
+    d. Select the file
+2. Make sure you have all the languages you will be using to solve problems installed on your local machine and referenced in your `$PATH`.
```
Comment (trigaten) [inline README.md:19]: should we mention installing them via plugin library?

File: keylogger-mvp/exec/exec.py [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+        return "Fizz"
+    elif n % 3 == 0:
+        return "Buzz"
+    else:
+        return ""
+
+
+    # divisors = []
+    # for i in range(1, n + 1):
+    #     if n % i == 0:
```
Comment (trigaten) [inline keylogger-mvp/exec/exec.py:14]: remove commented out code?

Comment (stefanhts) [reply keylogger-mvp/exec/exec.py:14]: This is code written during testing and I have added it to gitignore, not sure whether I should remove or not

File: README.md (hunk 1 above)
Comment (stefanhts) [reply README.md:19]: Not sure what you mean here

**Comment by stefanhts** [comment]
> Why does the vsx file get pushed?

It is the current build if we want to have it somewhere, not sure where we want to have the newest build

File: keylogger-mvp/exec/exec.py (hunk 2 above)
Comment (trigaten) [reply keylogger-mvp/exec/exec.py:14]: should almost always remove commented out code, I would encourage it.

**Comment by trigaten** [comment]
> > Why does the vsx file get pushed?
> 
> It is the current build if we want to have it somewhere, not sure where we want to have the newest build

The proper place to put that is likely in releases: https://github.com/code-keylogger/Extension/releases

File: README.md (hunk 1 above)
Comment (trigaten) [reply README.md:19]: You can download languages via the VSCode plugin library

File: keylogger-mvp/exec/exec.py (hunk 2 above)
Comment (stefanhts) [reply keylogger-mvp/exec/exec.py:14]: Yeah, I meant that this is generated code not code that does anything. Should it all be removed and the file just be empty?

File: README.md (hunk 1 above)
Comment (stefanhts) [reply README.md:19]: That won't work I don't think. We rely on the language functionality on their machine not any other plugins

**Comment by stefanhts** [comment]
> > > Why does the vsx file get pushed?
> > 
> > 
> > It is the current build if we want to have it somewhere, not sure where we want to have the newest build
> 
> The proper place to put that is likely in releases: https://github.com/code-keylogger/Extension/releases

I will put it there once this changeset is merged

**Pull request merged**